specification/machinelearningservices/resource-manager/readme.typescript.md

```diff
@@ -5,6 +5,7 @@ Please also specify `--typescript-sdks-folder=<path to root folder of your azure
 
 ``` yaml $(typescript)
 typescript:
+  title: AzureMachineLearningMgmtClient
```

Comment:
so the swagger title is: 
"title": "Azure Machine Learning Services",

therefore I think it should be MachineLearningServicesMgmtClient